server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -664,8 +651,48 @@ else if (ownerDto.getKey() != null) {
         }
     }
 
-    /**
+    /*
      * Populates the specified entity with data from the provided DTO.
+     */
+    private Owner lookupOwnerFromDto(OwnerDTO ownerDto) {
+        Owner owner = null;
+        if (ownerDto.getId() != null) {
+            // look up by ID
+            owner = this.ownerCurator.find(ownerDto.getId());
+        }
+        else if (ownerDto.getKey() != null) {
+            // look up by key
+            owner = this.ownerCurator.lookupByKey(ownerDto.getKey());
+        }
+        if (owner == null) {
```

Comment:
Whitespace between block structures. The lack of space makes them flow together at a glance.